ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/dps_interfaces.py

```diff
@@ -38,6 +38,7 @@ def dps_interfaces(self) -> list | None:
         # When needed - need a default value if different than IPv4
 
         if self.shared_utils.is_cv_pathfinder_router:
```

Comment:
We should not have this condition any longer. Maybe we need to adjust the default value for Dps interfaces to cover autovpn cases, since removing this would affect the defaults, but that is easier done elsewhere.

Reply:
Set it as default for now.